### PR TITLE
ARM template update

### PR DESCRIPTION
**Pull request by priyansh-sumo**
- Git Action
- ARM template Update

### CODE REVIEW

**Review by himanshu219 — CHANGES_REQUESTED** [review]
1. uncomment the  git actions for metrics arm template
2. There are some warnings related to api versions in the git checks for arm ttk

File: .github/workflows/arm-template-test.yml [hunk 1]
```diff
@@ -10,11 +9,53 @@ jobs:
...
+        id: metricfilestest
+        uses: microsoft/action-armttk@main
+        with:
+          workdir: EventHubs/src
+          fail_on_error: true
+          level: "info"
+
+      - name: Rename file 
+        run: |
+          rm -rf arm-ttk
```
Comment (himanshu219) [inline .github/workflows/arm-template-test.yml:29]: why do we need to do rm -rf arm-ttk ?

File: .github/workflows/arm-template-test.yml [hunk 2]
```diff
@@ -10,11 +9,53 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Test ARM Templates using arm-ttk
-        id: filestest
-        uses: aliencube/arm-ttk-actions@v0.3
+      - name: Rename file 
+        run: |
+          cp EventHubs/src/azuredeploy_metrics.json EventHubs/src/mainTemplate.json 
+          shopt -s extglob
```
[the remark this answers is not in the document]
Comment (priyansh-sumo) [reply .github/workflows/arm-template-test.yml]: removed

File: .github/workflows/arm-template-test.yml (hunk 1 above)
Comment (priyansh-sumo) [reply .github/workflows/arm-template-test.yml]: updated with metrix logic.

**Review by himanshu219 — APPROVED** [review]

**Pull request merged**